src/transformState.js

```diff
@@ -5,7 +5,48 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
You can make destructuring to get all the needed props of action (type, extraData, keysToRemove)